pkg/liqo-controller-manager/networking/external-network/remapping/ip_controller.go

```diff
@@ -86,7 +91,7 @@ func (r *IPReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
-	if ip.Status.IPMappings == nil || len(ip.Status.IPMappings) == 0 {
+	if len(ip.Status.IPMappings) == 0 {
```

Comment:
Why that?

Reply:
Because it is not necessary, len(nil) return 0